packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx

```diff
@@ -158,6 +182,15 @@ export const Virtualization = () => {
   const { targetDocument } = useFluent();
   const scrollbarWidth = useScrollbarWidth({ targetDocument });
 
+  const renderRow: RowRenderFunction<Item> = React.useCallback(
```

Comment:
nit: the confusion there is that it's not a function

I would call it `RowRenderer` and move to an upper scope

Reply:
Makes sense, done